api/v1alpha1/openapi.yaml

```diff
@@ -3000,10 +3000,14 @@ components:
       type: object
       required:
         - image
+        - imageDigest
       properties:
         image:
           type: string
           description: "Version of the OS image."
+        imageDigest:
+          type: string
+          description: "The digest of the OS image (e.g. sha256:a0...)"
```

Comment:
@celdrake WDYT? We want to do our best about reporting the digest in cases where there is a static tag like the latest and there could be divergence between the fleet.  This could be data that is show on hover or could just be used for debug,

Reply:
I agree that we should display this, as it's an important info important to know.
I'll discuss with Nir whether we want to display it by default or on hover, etc.